internal/provider/data_source_aci_bridge_domain.go

```diff
@@ -144,10 +148,6 @@ func (d *FvBDDataSource) Schema(ctx context.Context, req datasource.SchemaReques
 				Computed:           true,
 				DeprecationMessage: "Attribute 'relation_fv_rs_bd_to_nd_p' is deprecated, please refer to 'relation_to_neighbor_discovery_interface_policy.neighbor_discovery_interface_policy_name' instead. The attribute will be removed in the next major version of the provider.",
 			},
-			"relation_fv_rs_bd_to_profile": schema.StringAttribute{
-				Computed:           true,
-				DeprecationMessage: "Attribute 'relation_fv_rs_bd_to_profile' is deprecated, please refer to 'relation_to_route_control_profile.route_control_profile_name' instead. The attribute will be removed in the next major version of the provider.",
```

Comment:
The extra deprecation message referring to `route_control_profile_name` appears to have been removed. Was this expected?

Reply:
Decided to fully deprecate the attribute because during testing of legacy attributes I found that this attribute was never working for the resource. In order for the fvRsBDToProfile to resolve correctly both tnL3extOutName and tnRtctrlProfileName require to be provided, however the legacy attribute only allowed for tnRtctrlProfileName ( in the form of tDn where MoName gets stripped off ).